packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -200,20 +200,12 @@
  * This renders the component's active content.
  */
 {template .active}
+	{@param actionItems: list<?>}
```

Comment:
hey @jbalsas, I'm removing the typing here so that at https://github.com/liferay/clay/pull/1181/files#diff-54445d58e09b53d03b6d0558828824f6R136 can set the default value if not passed.

Since we are typing `actionItems` above, we can ensure that the values are passed correctly.